docs-2.0/nebula-cloud/5.solution/5.3.role-and-authority-management.md

```diff
@@ -0,0 +1,25 @@
+# Roles and authority management
+
+The role on the Cloud is different from that of the Nebula Graph. For more information, see [Roles in Cloud Solution](../4.user-role-description.md).
+
+|Roles in Cloud|Roles in Nebula Graph|
+|:---|:---|
+|OWNER|-|
+|ROOT|ROOT|
+|USER|ADMIN/DBA/GUEST/USER|
+
+## Root Management
+
+Only users with **OWNER** authority can manage **ROOT** users.
+
+On the Root Management page, OWNER can reset the ROOT users.
+
+Click `Reset`, enter the email address of the ROOT user to be updated, and click `Send Email` to send the email. After the designated user receives the confirmation email, click Confirm.
+
+## User Management
+
+Only users with **ROOT** authority can manage **USER** users.
+
+On the USER management page, ROOT can specify the user's authority to the graph space as `ADMIN`, `DBA`, `GUEST` ,or `USER`.
```

Comment:
Is this page `User Management` or `USER management`? And bold the keywords.

The ROOT user can grant roles in graph spaces to other users. Available roles are ADMIN, DBA, GUEST, and USER.